drivers/tests/TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -143,18 +153,16 @@ void test_detach(void)
 {
     LowPowerTicker ticker;
     bool ret;
-    const float ticker_time_s = 0.1f;
-    const uint32_t wait_time_ms = 500;
     Semaphore sem(0, 1);
 
-    ticker.attach(callback(sem_release, &sem), ticker_time_s);
+    ticker.attach(callback(sem_release, &sem), 100ms);
 
     sem.acquire();
 
     sem.acquire();
     ticker.detach(); /* cancel */
 
-    ret = sem.try_acquire_for(wait_time_ms);
+    ret = sem.try_acquire_for(500ms);
```

Comment:
Do we need the `ret` variable? Can we just use `TEST_ASSERT_FALSE(sem.try_acquire_for(500ms))`?

Reply:
Good shout - I have amended this and a similar case in mbed_drivers/ticker